.github/workflows/gitleaks.yaml

```
@@ -0,0 +1,20 @@
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  scan:
+    name: gitleaks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # fetch all history so multiple commits can be scanned
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)